modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/batch/BatchIndexingHelperImpl.java

```diff
@@ -42,19 +40,18 @@ public class BatchIndexingHelperImpl implements BatchIndexingHelper {
 
 	@Override
 	public int getBulkSize(String entryClassName) {
-		Optional<Integer> optional = Stream.of(
-			_reindexConfiguration.indexingBatchSizes()
-		).map(
-			line -> StringUtil.split(line, StringPool.EQUAL)
-		).filter(
-			pair -> pair.length == 2
-		).filter(
-			pair -> entryClassName.equals(pair[0])
-		).map(
-			pair -> GetterUtil.getInteger(pair[1])
-		).findAny();
+		for (String indexingBatchSize :
+				_reindexConfiguration.indexingBatchSizes()) {
 
-		return optional.orElse(Indexer.DEFAULT_INTERVAL);
+			String[] line = StringUtil.split(
```

Comment:
@jeffwu0724 `line` in the original code equals to `indexingBatchSize` here. This `String[]` should be called `pair`.